File: ifyoucanreadthis.rb
def to_nato(words)

  nato = {
    'A' => 'Alfa', 'B' => 'Bravo', 'C' => 'Charlie', 'D' => 'Delta', 'E' =>'Echo', 'F' =>'Foxtrot',
    'G' => 'Golf', 'H' => 'Hotel', 'I' => 'India', 'J' => 'Juliett', 'K' => 'Kilo', 'L' => 'Lima', 'M' => 'Mike',
    'N' => 'November', 'O' => 'Oscar', 'P' => 'Papa', 'Q' => 'Quebec', 'R' => 'Romeo', 'S' => 'Sierra', 'T' => 'Tango',
    'U' => 'Uniform', 'V' => 'Victor', 'W' => 'Whiskey', 'X' => 'Xray', 'Y' => 'Yankee', 'Z' => 'Zulu'
  }

nato_words = ''

words.delete!(' ')
words.upcase!


words.split('').each { |letter| !letter.match(/[^A-Za-z]/)? (nato_words << ' ' + nato[letter]) : (nato_words << ' ' + letter)  }

nato_words.strip!
end

puts(to_nato('If you can read'))

File: ifyoucanreadthis.py
def to_nato(words):
    nato = [
         'Alfa', 'Bravo', 'Charlie', 'Delta', 'Echo', 'Foxtrot',
        'Golf', 'Hotel', 'India', 'Juliett', 'Kilo', 'Lima', 'Mike',
        'November', 'Oscar', 'Papa', 'Quebec', 'Romeo', 'Sierra', 'Tango',
        'Uniform', 'Victor', 'Whiskey', 'Xray', 'Yankee', 'Zulu'
    ]

    words = words.replace(' ', '')
    words = words.lower()

    nato_words = ''

    for letter in words:
        if nato_words == '' and (ord(letter)-97) >= 0:
            nato_words = nato[ord(letter)-97]

        elif nato_words == '' and (ord(letter)-97) <= 0:
            nato_words = letter

        elif (ord(letter)-97) >= 0:
            nato_words = nato_words + ' ' + nato[ord(letter)-97]
        else:
            nato_words = nato_words + ' ' + letter

    return nato_words